### ISSUE TITLE
More minimalistic design

### ISSUE THREAD

**Issue by bitagoras** [microsoft/vscode#124429]
The current version of the notebook looks like this:
![Current_notebook_desing](https://user-images.githubusercontent.com/10660123/119226681-aa876880-bb0a-11eb-801e-3b5bf80ccc61.png)
The design has in my opinion too many bounding boxes and appears still overloaded. The tool buttons attract too much attention and appear misplaced when overlapping the editor field. Therefore I suggest the following more clean and simplistic design:
![Proposed_notebook_design](https://user-images.githubusercontent.com/10660123/119239066-cad51880-bb46-11eb-8803-0bc7f3b37b3b.png)
In summary:
- Line numbers are outside the cell and appear only at the selected cell.
- Controll icons are on the right and also appear only at the selected cell.
- The background for the Python code is dark as in standard Jupyter-Notebooks and white for the output (at aleast for the bright color theme). This is more optimal for plots that normally have white background.
- The execution counter (like [2]) is inside and grouped together with the timing information so that it is not confused with a line number.

**Comment by bitagoras** [comment]
Or even more puristic:
![Proposed_notebook_design2](https://user-images.githubusercontent.com/10660123/119329591-acc20200-bc85-11eb-8e31-4fadf071b332.png)
I think the design simplicity of VS Code is one of the major selling points that makes this project is so succesful. I think people would love to see this also in the notebook.

**Comment by bitagoras** [comment]
Well, actually, beeing a purist I have no justification for that round circles around the buttons:

![pure_symbols_crop](https://user-images.githubusercontent.com/10660123/119556953-e8e98580-bd9f-11eb-969c-ef354f178d30.png)

**Comment by bitagoras** [comment]
I close this issue since the design is meanwhile improved. The tool box is now more discreet on the right which is less annoying. I would still like to have less elements such as lines or buttons which are neither code nor output.
